telemetry/definitions/commonDefinitions.json

```diff
@@ -1585,6 +1591,15 @@
             ],
             "passive": true
         },
+        {
+            "name": "codewhisperer_autoSuggestionActivation",
+            "description": "State of cwspr auto-suggestion",
```

Comment:
Think something like "User activated or deactivated automated trigger for CodeWhisperer" is better